### DISCUSSION TITLE
Suricata logs -- can the agent pick these up?

### DISCUSSION THREAD

**Discussion by ihuxn4hi** [Security-Onion-Solutions/securityonion#11283] [2.4]
### Version

2.4.10

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

8

### RAM

32

### Storage for /

256G

### Storage for /nsm

250G

### Network Traffic Collection

other (please provide detail below)

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Hi there,

I'm currently migrating off of an Elastic Cloud deployment and had a question regarding Suricata log ingestion.

With the agent I'm using in Elastic Cloud (8.8.1), you just point it to the log file in the Integrations page:

<img width="798" alt="image" src="https://github.com/Security-Onion-Solutions/securityonion/assets/123765379/51b1ae8f-8334-48f8-9d75-87622f7ea115">

But in the integration with Security Onion, it appears you have to use a separate filebeat installation to pick up those logs:

<img width="754" alt="image" src="https://github.com/Security-Onion-Solutions/securityonion/assets/123765379/c9b5350d-5ff9-4f8a-aeb0-d08975ff4df2">

Other integrations, such as CrowdStrike, point to local log files:

<img width="795" alt="image" src="https://github.com/Security-Onion-Solutions/securityonion/assets/123765379/312ae065-8673-477b-a1f4-9b9d9892de7b">

So I'm not sure why it wouldn't be the same with Suricata as it definitely is in my Elastic Cloud deployment.

Thanks!

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by dougburks** [comment]
The best way to deploy Suricata is to deploy a Security Onion forward node as this automatically configures Suricata log ingestion (and many other things) for you:
https://docs.securityonion.net/en/2.4/architecture.html#forward-node

**Comment by ihuxn4hi** [comment]
Hi Doug!

Thank you so much for the reply. I definitely would do that if that was an option but I'm in AWS and mirrored ports ended up costing us an insane amount of money in a previous iteration. 

We've been able to successfully ingest Suricata logs by defining the log path with our Elastic Cloud deployment and it was as easy as configuring the output file, just as CrowdStrike is, as referenced above.  All other integrations are the same so I'm just curious why Suricata is so different.

Thanks again -- I really appreciate your help!
